pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_assigning-metadata-to-a-module.adoc

```diff
@@ -1,30 +1,31 @@
 [id="assigning-metadata-to-a-module_{context}"]
-= Assigning metadata to a module
+= Assigning metadata to {ContentTerm}
 
-Assign metadata to a module to enable search engine optimization (SEO). SEO improves the findability of content.
+[role="_abstract"]
+Assign metadata to {ContentTerm} to enable search engine optimization (SEO). SEO improves the findability of content. Assigned metadata also determines the URL at which the {ContentTerm} is published.
```

Comment:
determines -> defines, or 'helps define"

Reply:
changed to defines.